### PR TITLE
Implement sub recipe view of recipes and connect action.

### PR DESCRIPTION
**Pull request by Unengine**
### Description

SSIA

### How to test

1. Check if sub recipes and options are displaying properly when crafting.
2. Check if equipments and consumables can be crafted.
3. Set `isEditor` variable to `false` in `GameConfig.cs` and check if recipe unlocking works properly.

### CODE REVIEW

File: nekoyume/Assets/_Scripts/Helper/RecipeHelper.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+using Nekoyume.Model.Stat;
+using Nekoyume.TableData;
+using System.Linq;
+
+namespace Nekoyume.Helper
+{
+    public static class RecipeHelper
+    {
+        public static EquipmentItemSheet.Row GetResultItem(this EquipmentItemRecipeSheet.Row recipeRow)
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/Helper/RecipeHelper.cs:9]: How about this?

```suggestion
        public static EquipmentItemSheet.Row GetResultEquipmentRow(this EquipmentItemRecipeSheet.Row recipeRow)
```

File: nekoyume/Assets/_Scripts/Helper/RecipeHelper.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        {
+            return Game.Game.instance.TableSheets
+                .EquipmentItemSheet[recipeRow.ResultEquipmentId];
+        }
+
+        public static ConsumableItemSheet.Row GetResultItem(this ConsumableItemRecipeSheet.Row recipeRow)
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/Helper/RecipeHelper.cs:15]: How about this?

```suggestion
        public static ConsumableItemSheet.Row GetResultConsumableRow(this ConsumableItemRecipeSheet.Row recipeRow)
```

File: nekoyume/Assets/_Scripts/UI/Combination/SubRecipeView.cs [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+                toggle.onValueChanged.AddListener(value =>
+                {
+                    if (!value) return;
+                    AudioController.PlayClick();
+                    ChangeTab(innerIndex);
+                });
+            }
+
+            combineButton.onClick.AddListener(() =>
+                CombinationActionSubject.OnNext(_selectedRecipeInfo));
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/UI/Combination/SubRecipeView.cs:87]: Any `AudioController.PlayClick();`?

File: nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs [hunk 4]
```diff
@@ -142,7 +142,7 @@ public void SetData(TViewModel model, bool isConsumable)
             }
             else
             {
-                row = Game.Game.instance.TableSheets.ItemSheet.Values
+                row = Game.Game.instance.TableSheets.EquipmentItemSheet.Values
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs:145]: This change make some problems I guess. The `ItemView` module support not only equipment.

Comment (Unengine) [reply nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs:145]: This method is used only in combination, but I think i should extend this to support both equipment and other item. Gonna fix it!

File: nekoyume/Assets/_Scripts/UI/Combination/SubRecipeView.cs (hunk 3 above)
Comment (Unengine) [reply nekoyume/Assets/_Scripts/UI/Combination/SubRecipeView.cs:87]: I forgot. Gonna fix it!

**Review by hskim881028 — APPROVED** [review]

File: nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs (hunk 4 above)
Comment (boscohyun) [reply nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs:145]: The `EqiupmentItemSheet` does not contains any of `Consumable` rows. Is it OK?

Comment (Unengine) [reply nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs:145]:
```
            if (isConsumable)
            {
                row = Game.Game.instance.TableSheets.ConsumableItemSheet.Values
                    .FirstOrDefault(r => r.Id == model.ItemBase.Value.Id);
            }
            else
            {
                row = Game.Game.instance.TableSheets.EquipmentItemSheet.Values
                    .FirstOrDefault(r => r.Id == model.ItemBase.Value.Id);
            }
 ```
 
 It is already handled like this.

Comment (boscohyun) [reply nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs:145]: !!!! Oh I missed. 😅

**Review by boscohyun — APPROVED** [review]

**Pull request merged**